doc/sphinx/openquake.hazardlib.rst

```diff
@@ -12,8 +12,6 @@ Subpackages
     openquake.hazardlib.mfd
     openquake.hazardlib.scalerel
     openquake.hazardlib.source
-    openquake.hazardlib.source_reader
-    openquake.hazardlib.logictree
```

Comment:
This 2 lines should not be removed. Instead openquake.commonlib.source_reader and openquake.commonlib.logictree should be removed.

Reply:
They are already present below as automodules